src/nvim/ex_docmd.c

```diff
@@ -1156,13 +1156,24 @@ static void get_wincmd_addr_type(char_u *arg, exarg_T *eap)
     case Ctrl_I:
     case 'd':
     case Ctrl_D:
-      /* window size or any count */
+    case Ctrl_Z:
+    case 'z':
+    case 'P':
+    case 't':
+    case Ctrl_T:
+    case 'b':
+    case Ctrl_B:
+    case 'p':
+    case Ctrl_P:
+    case '=':
+    case CAR:
+      // window size or any count
       eap->addr_type = ADDR_LINES;
```

Comment:
`#define ADDR_LINES              0`

Reply:
This is different semantically that "no count", which we've now lost as a hint/documentation.

This particular PVS warning is really annoying. I wish we could find a way to disable some warnings project-wide.

In this case I don't think this is a good change.

Reply:
ack. will add ignore to this one.